coderd/aitasks.go

```diff
@@ -832,7 +832,7 @@ func (api *API) authAndDoWithTaskAppClient(
 	appURL := app.Url.String
 	if appURL == "" {
 		return httperror.NewResponseError(http.StatusInternalServerError, codersdk.Response{
-			Message: "Task sidebar app URL is not configured.",
+			Message: "Task app URL is not configured.",
```

Comment:
Could this be enforced provider-side? 🤔 It's unfortunate to let it be a runtime error that basically turns the task defunct 😅.

Reply:
I'm not sure if you can do cross-resource validation in Terraform, and you define the `url` in the `workspace_app` resource :(

Reply:
Then validation on provisioner side would perhaps be an option.